talentprotocol/mvp
# Change discovery page banner to twitter

## Summary
Change banner to twitter related one

## How to test?

See Discovery page's banner and try to click on button, it should open a new tab with Talent Protocol's twitter
